app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -56,6 +56,8 @@ def build_access_request
       end
 
       def access_request_params
+        #This needs to be modified so requester_email can't be changed by the frontend
+        # or is that not actually an issue?
```

Comment:
The `access_request_params` appears to only be used by the `create` action, so I think that's ok.